src/include/baselib/messaging/BrokerErrorCodes.h

```diff
@@ -75,6 +75,16 @@ namespace bl
                 return false;
             }
 
+            static auto tryGetExpectedErrorMessage( SAA_in const bl::eh::error_code& ec ) NOEXCEPT -> const std::string
+            {
+                if( isExpectedErrorCode( ec ) )
+                {
+                    return "The server is currently unavailable";
```

Comment:
1. as per above this should be static class string otherwise you will be making unnecessary copy each time and the method in this case can't be NOEXCEPT (as the std::string constructor can throw)
2. Why return a generic message for all error codes? It is best to have a switch on the known error codes and return the respective error code specific message. Also for the first two standard messages the error code message should come from the error code class itself (ec.message())